2089_find_target_indices.py

```diff
@@ -0,0 +1,43 @@
+# Time complexity O(n)
+# Space complexity O(1)
+from typing import List
+
+
+class Solution:
+    
+    def targetIndices(self, nums: List[int], target: int) -> List[int]:
+        nums.sort()
+        
+        def binarySearchLeft(nums, target):
+            left, right = 0, len(nums) - 1
+            while left <= right:
+                mid = (left + right) // 2
```

Comment:
mid = (left+right) // 2
If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority (see PEP8).